File: fizzbuzz/fizzbuzz.go
package main
import "fmt"

func main() {
  fmt.Printf("Fizz Buzz\n")
  for i:=1; i<=15; i++ {
    if (i % 3 != 0) && (i % 5 != 0) {
      fmt.Printf("%d", i)
    } 
    if (i % 3 == 0) {
      fmt.Printf("Fizz")
    } 
    if (i % 5 == 0) {
      fmt.Printf("Buzz")
    }
    fmt.Printf(" ")
  }
}

File: fizzbuzz/fizzbuzz.rb
output=""
(1..15).each do |x|
  if (x % 3 == 0)
    output+="Fizz"
  end
  if (x % 5 == 0)
    output += "Buzz"
  end
  if (x % 3 != 0 && x % 5 != 0)
    output +=x.to_s
  end
  output+=" "
end
puts output;
